ibm_catalog.json

```
@@ -271,36 +282,37 @@
 							"key": "secret_manager_public_engine_enabled",
 							"type": "boolean",
 							"default_value": false,
-							"description": "Set this to true to configure a Secrets Manager public certificate engine for an existing Secrets Manager instance. If set to false, no public certificate engine will be configured for your instance.",
+							"description": "Whether to configure a Secrets Manager public certificate engine for an existing Secrets Manager instance. If `false`, no public certificate engine is configured for your instance.",
 							"required": false
 						},
 						{
 							"key": "cis_id",
 							"type": "string",
 							"default_value": "__NULL__",
-							"description": "Cloud Internet Service ID.",
+							"description": "Cloud Internet Service ID. Required if `secret_manager_public_engine_enabled` is set to true.",
 							"required": false
 						},
 						{
 							"key": "ca_name",
 							"type": "string",
-							"default_value": "cert-auth",
-							"description": "The name of the certificate authority for Secrets Manager.",
+							"default_value": "__NULL__",
+							"description": "The name of the certificate authority for Secrets Manager. Required if `secret_manager_public_engine_enabled` is set to true.",
 							"required": false
 						},
 						{
 							"key": "dns_provider_name",
 							"type": "string",
-							"default_value": "certificate-dns",
-							"description": "The name of the DNS provider for the public certificate secrets engine configuration.",
+							"default_value": "__NULL__",
+							"description": "The name of the DNS provider for the public certificate secrets engine configuration. Required if `secret_manager_public_engine_enabled` is set to true.",
 							"required": false
 						},
 						{
 							"key": "acme_letsencrypt_private_key",
 							"type": "string",
 							"default_value": "__NULL__",
-							"description": "The private key generated by the ACME account creation tool.",
-							"required": false
+							"description": "The private key generated by the ACME account creation tool. For more information, see [documentation](https://cloud.ibm.com/docs/secrets-manager?topic=secrets-manager-prepare-order-certificates)",
```

```suggestion
							"description": "The private key generated by the ACME protocol. For more information, see [Preparing to order public certificates](https://cloud.ibm.com/docs/secrets-manager?topic=secrets-manager-prepare-order-certificates).",
```

The docs refer to it as a protocol instead of a tool, so suggesting that here.